BentoKit/BentoKit/Components/TitledDescription.swift

```diff
@@ -43,7 +43,6 @@ public extension Component {
             image: Property<BentoKit.ImageOrLabel>? = nil,
             accessory: Accessory = .chevron,
             badgeIcon: UIImage? = nil,
-            isEnabled: Bool = true,
```

Comment:
Why was isEnabled removed? Was it not being used by anything?

Reply:
not used as flagged by @mluisbrown